### PR TITLE
chore!: test suite updated

### PR DESCRIPTION
**Pull request by waxb**
- chore: changelog updated
- chore: test suite updated

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -79,17 +79,17 @@ If everything looks good, they will merge the code and release a new version whi
...
-
 [github flow]: https://guides.github.com/introduction/flow/
 [codeowners]: https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
 [fork]: https://help.github.com/en/github/getting-started-with-github/fork-a-repo
-[create a new branch]: https://guides.github.com/introduction/flow/
-[readme driven development]: https://tom.preston-werner.com/2010/08/23/readme-driven-development.html
 [commit]: https://help.github.com/en/desktop/contributing-to-projects/committing-and-reviewing-changes-to-your-project
+[create a new branch]: https://guides.github.com/introduction/flow/
 [create a pull request]: https://help.github.com/articles/creating-a-pull-request/
+
```
Comment (soerenmartius) [inline CONTRIBUTING.md:88]: this isn't the latest version of `CONTRIBUTING.md`, please see https://github.com/mineiros-io/terraform-module-template/pull/164 for details

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Test suite updated
```
Comment (soerenmartius) [inline CHANGELOG.md:10]:
```suggestion

### Added

- Add unit tests
```

**Review by soerenmartius — CHANGES_REQUESTED** [review]

File: README.tfdoc.hcl [hunk 3]
```diff
@@ -168,7 +168,7 @@ section {
         }
 
         variable "access" {
-          type        = list(any)
+          type        = any
```
Comment (mariux) [inline README.tfdoc.hcl:171]:
```suggestion
          type        = list(access)
```

File: README.tfdoc.hcl [hunk 4]
```diff
@@ -288,8 +288,7 @@ section {
         }
 
         variable "iam" {
-          type        = list(any)
-          readme_type = "list(iam)"
+          type        = any
```
Comment (mariux) [inline README.tfdoc.hcl:291]:
```suggestion
          type        = list(iam)
```

**Review by mariux — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (mariux) [reply CHANGELOG.md:10]: not resolved yet.

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
```
Comment (mariux) [inline CHANGELOG.md:11]:
```suggestion
### Added

- Add unit tests

```

**Review by mariux — CHANGES_REQUESTED** [review]

**Review by mariux — CHANGES_REQUESTED** [review]
- tests are failing -> not ready for review
- please mark the breaking commit not only the PR.

File: CHANGELOG.md (hunk 5 above)
Comment (mariux) [reply CHANGELOG.md:13]: not resolved yet.

File: CHANGELOG.md [hunk 6]
```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Test suite updated
+
+### Removed
+
+- BREAKING: Support for provider 3.x
```
Comment (mariux) [inline CHANGELOG.md:16]:
```suggestion
- BREAKING: Support for provider 3.x removed
```

**Review by mariux — COMMENTED** [review]
There are still unresolved change requests ;)

File: CHANGELOG.md [hunk 7]
```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Add unit tests
+
+### Removed
+
+- BREAKING: Support for provider 3.x
```
Comment (mariux) [inline CHANGELOG.md:16]:
```suggestion
- BREAKING: Remove support for Terraform versions before v1.0
- BREAKING: Remove support for Terraform Google Provider versions before v4.0
```

**Review by mariux — CHANGES_REQUESTED** [review]

**Review by mariux — APPROVED** [review]
lgtm thanks ;)

**Review by soerenmartius — APPROVED** [review]
lgtm

**Pull request merged**